config/application.rb

```diff
@@ -71,5 +71,8 @@ class Application < Rails::Application
 
     # Include full precision of timestamps in JSON responses
     config.active_support.time_precision = 6
+
+    I18n.available_locales = %i[en cy]
+    I18n.default_locale = :en
```

Comment:
Using the `I18n` variables makes sense.

The values are used across the rails app though. I wonder if we will ever want to separate the concept of available locales for the Forms admin app (probably only ever English?) vs. the available locales for the forms our users create?